src/ape/plugins/__init__.py

```diff
@@ -122,6 +123,15 @@ class PluginManager:
     _unimplemented_plugins: List[str] = []
 
     def __init__(self) -> None:
+        # NOTE: This is a hack to make 'ape -v debug' correctly debug failing plugins.
```

Comment:
Other things I tried before this:

* Override something in the `click.Multicommand` we are using and force-load the argument there. But, based on where this all happens in the click flow, that did not seem possible without hacky measures (as far as I could tell)
* Put this same logic at the top of the main `__init__.py` script... that did not work either. Odd.

I am open to suggestions!

It is really nice being able to debug failing plugins now!

Reply:
Was able to put it in the logging module itself, so now when you request the logger, it sets